backend/app/routers/authorization.py

```diff
@@ -336,65 +350,50 @@ async def remove_dataset_user_role(
         raise HTTPException(status_code=404, detail=f"Dataset {dataset_id} not found")
 
 
-@router.get("/datasets/{dataset_id}/users_and_roles", response_model=List[UserAndRole])
-async def get_dataset_users_and_roles(
+@router.get("/datasets/{dataset_id}/roles")  # , response_model=DatasetRoles
```

Comment:
if you don't need the `response_model` maybe just remove it?

Reply:
added back thanks, it is accurate so not sure why i had it commented out